force-app/main/default/lwc/edaSettingsNavigationSection/edaSettingsNavigationSection.html

```diff
@@ -0,0 +1,69 @@
+<template>
+    <div class="slds-col slds-size_12-of-12">
+        <!--Section Header-->
+        <h2
+            id={viewModel.id}
+            class="slds-text-heading_small slds-p-left_large slds-p-top_medium slds-p-bottom_xx-small"
+        >
+            <template if:false={viewModel.page}> {viewModel.label} </template>
+            <template if:true={viewModel.page}>
+                <a
+                    href="javascript:void(0);"
+                    class="slds-text-color_default eda-nav-menu-item_link"
+                    onclick={handleNavigationClick}
+                    aria-current={viewModel.isActive}
+                    data-qa-locator={qaLocator}
+                    >{viewModel.label}</a
+                >
+            </template>
+        </h2>
+
+        <!--Header Only Menu Items-->
+        <template if:true={hasMenuItems}>
+            <!--Base State-->
+            <template if:false={viewModel.isActive}>
+                <ul aria-describedby={viewModel.id}>
+                    <c-eda-settings-navigation-menu-items
```

Comment:
I think the child component needs to be wrapped in <li></li> since it is part of an unordered list element